distribution.properties

```diff
@@ -22,7 +22,7 @@ dbbench-ver=1.3
 jssh-terminal-ver=1.13
 log-viewer-ver=2.0.5
 pogo-ver=9.6.18
-rest-server-ver=1.9
+rest-server-ver=1.11
```

Comment:
@Ingvord , could you please also update the README?
https://github.com/tango-controls/TangoSourceDistribution/blob/prepare-9.3.3/assets/README

Reply:
Done in f409a5a